bindata/network/multus/multus.yaml

```diff
@@ -84,6 +84,38 @@ data:
       echo "Failed to copy files in ${sourcedir} to $DESTINATION_DIRECTORY"
       exit 1
     fi
+  allowlistcopy.sh: |-
+    #!/bin/bash
+    set -e
+
+    ALLOWLIST_CONF=/entrypoint/allowlist.conf
+    DESTINATION_DIRECTORY=/host/etc/cni/tuning/
+
+    cp -f $ALLOWLIST_CONF $DESTINATION_DIRECTORY
+
+    if [ $? -eq 0 ]; then
+      echo "Successfully copied tuning-cni config $ALLOWLIST_CONF to $DESTINATION_DIRECTORY"
+    else
+      echo "Failed to copy tuning-cni config $ALLOWLIST_CONFto $DESTINATION_DIRECTORY"
+      exit 1
+    fi
+  allowlist.conf: |-
+    ^net.ipv4.conf.IFNAME.send_redirects$
+    ^net.ipv4.conf.IFNAME.accept_source_route$
+    ^net.ipv6.conf.IFNAME.accept_source_route$
+    ^net.ipv4.conf.IFNAME.accept_redirects$
+    ^net.ipv6.conf.IFNAME.accept_redirects$
+    ^net.ipv4.conf.IFNAME.secure_redirects$
+    ^net.ipv4.conf.IFNAME.log_martians$
+    ^net.ipv4.conf.IFNAME.rp_filter$
+    ^net.ipv6.conf.IFNAME.forwarding$
+    ^net.ipv4.conf.IFNAME.arp_notify$
+    ^net.ipv4.conf.IFNAME.arp_accept$
+    ^net.ipv4.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+    ^net.ipv4.conf.IFNAME.disable_policy$
+    ^net.ipv6.conf.IFNAME.disable_policy$
```

Comment:
would it make sense to order this somehow ? at least have all the ipv4 knobs listed together, then the ipv6 ones.

Reply:
np, done

Reply:
I actually like it better that each sysctl pair is togther
maybe even move to regex `"net.ipv[4|6].conf.IFNMAE..."`
to show that we are adding both variants

Reply:
But what about the odd one out - i.e. the ones without matches ? Like `send_redirects` , `log_martians`, `rp_filter`, etc ?...

Reply:
externally to this PR, I think we should complete the work on their safety and add them
but meanwhile,
with my ordering suggestion it will be much easier to call them out (ie- see they are different)

Reply:
... and listing them by IP family allows the reader to easily understand what knobs are available for each family - as I see it, it nearly becomes documentation. 

I won't fight for this, it can happen either way.

Reply:
.. meanwhile, until all interested parties reach an agreement on the order, I'm leaving the sorting as is :)
The list was further reduced. More sysctls will be added when certified as safe.